suites/tipset/message_dedup.go

```diff
@@ -24,71 +24,81 @@ func TestBlockMessageDeduplication(t *testing.T, factory state.Factories) {
 	t.Run("apply a single BLS message", func(t *testing.T) {
 		td := builder.Build(t)
 		defer td.Complete()
-		blkBuilder := drivers.NewTipSetMessageBuilder(td)
+		tipB := drivers.NewTipSetMessageBuilder(td)
+		blkB := drivers.NewBlockBuilder(td.ExeCtx.Miner)
```

Comment:
It is illegal for a tipset to have more than one block with the same miner, so the miner must be moved out of the test drive execution context, and prepared explicitly.

This change could come in a follow-up.